python-modules/cis_publisher/cis_publisher/hris.py

```diff
@@ -134,8 +134,14 @@ def deactivate_users(self, publisher, profiles, report):
                 # Convert as needed to a dict
                 try:
                     profile = profile.as_dict()
-                except (TypeError, AttributeError):
-                    pass
+                except (TypeError, AttributeError) as e:
```

Comment:
any clue why this can fail?

Reply:
changes in API replies from CIS to the publisher